### DISCUSSION TITLE
What python versions are supported? Is Python 3.11 supported?

### DISCUSSION THREAD

**Discussion by georgedahl** [thomaspinder/GPJax#334] [Q&A] [answered]
What python versions are currently supported?

**Comment by daniel-dodd** [comment, marked as answer]
Thanks for the question @georgedahl, as detailed in our [`pyproject.toml`](https://github.com/JaxGaussianProcesses/GPJax/blob/ce4090477fd11dde0403351e97a3f37a450faf00/pyproject.toml#L18C1-L18C23), versions `0.6.x` of `GPJax` support Python versions `3.8` to `3.11`.
